### DISCUSSION TITLE
flink-doris-connector 写入问题

### DISCUSSION THREAD

**Discussion by cfwldym** [apache/doris#17376] [A - General / Q&A]
使用 flink-doris-connector-1.6.0 版本1.3.0 ，写入一张表时，表的  datetime default current_timestamp 字段是NULL值， 但是用ssb的导数脚本导入 datetime default current_timestamp字段却是生效的，但是两者的本质都是Stream Load吧

**Comment by LemonLiTree** [comment]
ssb里面应该是自带了stream load的参数：-H "columns: update_time=current_timestamp()" ，你也可以在flink 导入也加这个配置，就可以了
